src/components/Footer/Footer.module.scss

```diff
@@ -0,0 +1,16 @@
+.footer {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #ffe9e0;
+  color: #230903;
+  height: 50px;
+  width: 100vw;
+  padding: 5px 10px;
+  text-align: left;
+  font-family: sans-serif, Calibri;
+  font-size: 14px;
+  font-weight: 300;
```

Comment:
I would recommend picking 3-4 weights in a normal project. You usually need a different font for each weight and that can get quickly out of hand. 

Also, some visual consistency is nice. This is the first font-weight I see, so it is completely fine. I am just trying to give head ups and some recommendations